documentation/source/user_section/tutorials/gm-wm/improving-registration-with-gm-seg/gm-informed-t2s-template-registration.rst

```diff
@@ -0,0 +1,54 @@
+################################################################
+
+First, we will need to generate the warping fields that represent the transformation between the T2* space and the PAM50 template space. Once we have that transformation, we can then use it to improve the registration results of MT data.
+
+This registration step is similar to that of previous tutorials. However, a notable difference is that rather than using the full spinal cord segmentations for the arguments ``-iseg`` and ``-dseg``, we use the white matter segmentation. This is because its interior perimeter outlines the gray matter shape, while its exterior perimeter outlines the shape of the spinal cord itself, which better informs the registration algorithm.
+
+.. code:: sh
+
+   sct_register_multimodal -i "${SCT_DIR}/data/PAM50/template/PAM50_t2s.nii.gz" \
+                           -iseg "${SCT_DIR}/data/PAM50/template/PAM50_wm.nii.gz" \
+                           -d t2s.nii.gz \
+                           -dseg t2s_wmseg.nii.gz \
+                           -initwarp ../t2/warp_template2anat.nii.gz \
+                           -initwarpinv ../t2/warp_anat2template.nii.gz \
+                           -param step=1,type=seg,algo=rigid:step=2,type=seg,algo=bsplinesyn,slicewise=1,iter=3 \
+                           -qc ~/qc_singleSubj
+
+:Input arguments:
+   - ``-i`` : Source image. Here, we select the T2* version of the PAM50 template.
+   - ``-iseg`` : Segmentation for the source image. Here, we use the PAM50 segmented white matter volume, rather than the spinal cord volume. This allows us to account for both the cord and the gray matter shape.
+   - ``-d`` : Destination image.
+   - ``-dseg`` : Segmentation for the destination image. Here, we use the white matter segmentation for the same reasons as ``-iseg``.
+   - ``-initwarp`` : Warping field used to initialize the source image. Here, we supply the ``warp_template2anat.nii.gz`` file from the previous T2 registration. (See: :ref:`template-registration`)
+      - TODO: I'm concerned that the usage of the ``-initwarp`` flag is not clear enough. For users who are following this tutorial standalone, they may not have the full context of the previous tutorial, so the usage of ``-initwarp`` may appear to "come out of nowhere". But, even if they have the context, reusing intermediate results raises some questions:
+          - Is ``-initwarp`` optional or mandatory?
+          - What should the command look like without ``-initwarp``?
+          - Will the registration results be significantly worse without ``-initwarp``?
+          - In what cases should ``-initwarp`` be used, and in what cases can they perform registration directly?
+   - ``-initwarpinv``: Warping field used to initialize the destination image. Here, we supply the inverse warping field, ``warp_anat2template.nii.gz`` from the previous T2 registration. (See: :ref:`template-registration`)
+      - TODO: Same concerns as above.
```

Comment:
I had some questions to try and pin down the motivations for why we are supplying the T2 warping fields to the T2* WM-informed registration.

Reply:
> I think that registration to PAM50 template with T2 is much more precise since it uses vertebral labeling etc (with `sct_register_to_template`), which is not the case with T2s. That would be the rational for using the warping field from T2 to PAM50 template for T2s registration

That makes sense!

Just to be thorough, though, I'd want to ask the same question I did for [the dMRI case](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3493#discussion_r687854616): "Could we performing vertebral labeling for the T2* data, then use `sct_register_to_template` directly?"

Or, does T2* data have a limitations like dMRI data, as explained by @jcohenadad: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3493#discussion_r687911379

Reply:
> Or, does T2* data have a limitation like dMRI data, as explained by @jcohenadad: #3493 (comment)

yes it does, because these data are also acquired axially with thick slices and intervertebral discs are poorly visible